torch/_inductor/codegen/rocm/rocm_benchmark_request.py

```diff
@@ -39,12 +40,15 @@ def __init__(
         self.hash_key: str = ""
         self.source_file: str = ""
         self.hash_key, self.source_file = ROCmCodeCache.write(self.source_code, "so")
+        self.generate_runner = generate_runner
 
     def precompile(self):
         # Prepopulate code cache
         # may happen in separate Threadpool
         log.debug("Precompiling %s", self)
         ROCmCodeCache.compile(self.source_code, "so")
+        if self.generate_runner:
+            ROCmCodeCache.compile(self.source_code, "exe")
```

Comment:
nit: also feeling that just referencing `config.rocm.generate_test_runner` would be cleaner